website/www/site/data/capability_matrix.yaml

```diff
@@ -393,7 +393,7 @@ capability-matrix:
             - class: dataflow
               l1: "Partially"
               l2: non-merging windows
-              l3: State is supported for non-merging windows. The MapState and SetState classes are supported with streaming pipelines that use Streaming Engine and version 2.58.0 or later of the Java SDK. SetState and MapState are not supported for pipelines that use Runner v2.
+              l3: State is supported for non-merging windows. The MapState and SetState classes and the MultimapState interface are supported in the following scenarios: Java pipelines that don't use Streaming Engine; Java pipelines that use Streaming Engine and version 2.58.0 or later of the Java SDK. SetState, MapState, and MultimapState are not supported for pipelines that use Runner v2.
```

Comment:
MapState, SetState and MultimapState are all interfaces so I think that the class/interface distinction is confusing.

Maybe "The MapState, SetState, and MultimapState state types are supported"